math_quiz.py

```diff
@@ -13,7 +13,7 @@ def build_questions():
 
     Each dictionary defines a first and second term, and math operator method and symbol
```

Comment:
@sophiasevim, you missed a spot. Would you mind adding a period at the end of line 14?